datapipelines/src/main/kotlin/io/customer/sdk/CustomerIO.kt

```diff
@@ -48,7 +60,7 @@ class CustomerIO private constructor(
     androidSDKComponent: AndroidSDKComponent,
     override val moduleConfig: DataPipelinesModuleConfig,
     overrideAnalytics: Analytics? = null
-) : CustomerIOModule<DataPipelinesModuleConfig>, DataPipelineInstance() {
+) : CustomerIOModule<DataPipelinesModuleConfig>, DataPipelineInstance(), MigrationProcessor {
```

Comment:
can we probably create a separate class for it? `CustomerIO` class is already too big 😅

Reply:
I was thinking the same initially, let me try to decouple it and share if I see any challenges in it.

Reply:
I moved code to `TrackingMigrationProcessor`. Let me know how if it looks good to you now.